apps/aechannel/src/aesc_fsm.erl

```diff
@@ -854,6 +854,15 @@ channel_closed(cast, {?MIN_DEPTH_ACHIEVED, ChainId, ?WATCH_CLOSED, TxHash},
                                         , tx_hash = TxHash % same tx hash
                                         } } = D) ->
     close(closed_confirmed, D);
+channel_closed(cast, {?CHANNEL_CHANGED, #{ tx_hash := TxHash
```

Comment:
This is an actual bug fix in the FSM. If the `channel_close_mutual_tx` actually ends up rejected in a (micro) fork only to be included later on in an another microblock, we should not "Panic and terminate" (for the panic and terminate, please chech the next function head)